CI/integration_tests/test_graph01.py

```diff
@@ -0,0 +1,96 @@
+"""
```

Comment:
Can you put an underscore before the 01 i.e. test_graph_01

Reply:
still stands.

Reply:
solved in 2175398a95c5813644aef5da0d41bc8ce2df9d21